src/sqlancer/StatementReducer.java

```diff
@@ -84,4 +123,4 @@ private void printQueries(List<Query<C>> statements) {
         System.out.println("===============================");
     }
 
-}
+}
```

Comment:
tiny issue: it's better to have an extra empty line at the end of file.

Detail: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file